packages/form-js-viewer/test/spec/render/components/FormField.spec.js

```diff
@@ -246,23 +246,6 @@ describe('FormField', function() {
     });
 
 
-    it('should use initial data to check condition', function() {
```

Comment:
Should we not keep this around, i.e. as an integration test in `Form.spec`?

Reply:
It's already covered in Form.spec: https://github.com/bpmn-io/form-js/pull/403/files#diff-f7f5270dd58d99d4b13f56770a7bcd44bee6b5d7a2f7d467fc4cf29b8f6d5093R1020-R1049